src/OrderBook.tsx

```diff
@@ -120,7 +120,12 @@ const renderList = (
         };
 
         return (
-          <li className={`${stylePrefix}__list-item`} key={price} style={rowStyle}>
+          <li
+            className={`${stylePrefix}__list-item`}
+            data-testid={price}
```

Comment:
I'm curious if this is strictly needed. Is it possible to use the `queryByRole` functions to pick up these `li` elements?

Reply:
switched it to use the queryAllByRole

Reply:
Nice!